Userland/Demos/Mouse/main.cpp

```diff
@@ -148,14 +151,25 @@ class MainFrame final : public GUI::Frame {
         start_timer(500);
     }
 
+    void doubleclick_event(GUI::MouseEvent& event) override
+    {
+        m_double_click_buttons |= event.button();
+        update();
+        if (has_timer())
+            stop_timer();
+        start_timer(500);
+    }
+
 private:
     unsigned m_buttons;
     unsigned m_wheel_delta_acc;
     bool m_show_scroll_wheel;
+    unsigned m_double_click_buttons;
```

Comment:
This variable name is also not clear to me. Perhaps m_double_clicked_button would be more appropriate, as from what I understand it holds the value for which button was clicked.